vendor/knative.dev/operator/pkg/reconciler/knativeeventing/eventing_tls.go

```diff
@@ -44,7 +46,11 @@ func (r *Reconciler) handleTLSResources(ctx context.Context, manifests *mf.Manif
 	// Delete TLS resources (if present)
 	toBeDeleted := manifests.Filter(tlsResourcesPred)
 	if err := toBeDeleted.Delete(mf.IgnoreNotFound(true)); err != nil && !meta.IsNoMatchError(err) {
-		return fmt.Errorf("failed to delete TLS resources: %v", err)
```

Comment:
I don't know how this worked before, if CRDs are not installed, this will error out and not filter out the eventing manifests (below). With this, no resources in knative-eventing were created in `make test-e2e`. The current change is only here to test, we should do this upstream, if it works. cc @creydr @matzew @pierDipi @maschmid 

related to https://issues.redhat.com/browse/SRVKE-1595

Reply:
Note to myself, maybe the new client returns a different error that is not handled by `IsNoMatchError`? In debug I see it is a `groupDiscoveryError`.

Reply:
https://issues.redhat.com/browse/SRVKE-1595 is just a warning not affecting functionality. 

I'm not sure I understand what is the real issue here

Reply:
If the error is different with the new clients, I think we need to understand why and if it is intended or not